src/components/core/MeshUIComponent.js

```diff
@@ -266,6 +266,10 @@ export default function MeshUIComponent( Base = class {} ) {
             return (this.hiddenOverflow === undefined) ? DEFAULTS.hiddenOverflow : this.hiddenOverflow;
         }
 
+        getLetterSpacing() {
+          return (this.letterSpacing === undefined) ? DEFAULTS.letterSpacing : this.letterSpacing;
+        }
+
```

Comment:
Here I think the property could be fetched from a parent with `_getProperty`, but I will discuss this more in length later